htdocs/stripe/admin/stripe.php

```diff
@@ -104,6 +104,10 @@
 	if (!$result > 0) {
 		$error++;
 	}
+	$result = dolibarr_set_const($db, "STRIPE_LOCATION", GETPOST('STRIPE_LOCATION', 'alpha'), 'chaine', 0, '', $conf->entity);
```

Comment:
You should add  
if (GETPOSTISSET('STRIPE_LOCATION'))
so when switching from debug and not debug, updating setup will not clear setup.

Reply:
@eldy done today :)